src/UserAgreement/Quiz.jsx

```diff
@@ -0,0 +1,193 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Question from './Question';
+import QuizSummary from './QuizSummary';
+import './Quiz.less';
+
+const getQuestionSectionId = idx => `Q${idx}`;
+
+const makeDefaultState = (questions) => {
+  const notDone = [];
+  questions.forEach(
+    (item, i) => {
+      notDone.push(i);
+    },
+  );
+  return {
+    notDone,
+    rights: [],
+    wrongs: [],
+    currentSection: getQuestionSectionId(1),
+  };
+};
+
+/**
+ * Little quiz component - roperites: questionList, title, onSubmit
```

Comment:
add `description` to comments?  or just remove the comments, I think proTypes is already self-explained enough.